README.md

```diff
@@ -105,6 +105,41 @@ Use `#pragma mark -` to categorize methods in functional groupings and protocol/
 - (NSString *)description {}
 ```
 
+## Header Imports
+
+#import statements should be kept to a minimum, especially in header files. Within header files use @class and @protocol where possible to avoid importing those dependencies.
+
+Headers should at least, directly or indirectly, import <Foundation/Foundation.h> (any common SDK framework header will import foundation indirectly).
+
+Framework imports should always import the framework umbrella header, not any of it individual header files.
+
+#import statements should be organized as follows, in sections separated by a white line:
+* System imports in alphabetical order (low level UNIX headers, if any).
+* System framework imports in alphabetical order (i.e. <Cocoa/Cocoa.h>)
+* Project framework imports in alphabetical order
+* Local imports in alphabetical order
```

Comment:
While grouping headers in sections is a reasonable (and simple) thing to do, alphabetical order has no obvious advantages comparing to the overhead needed to manually do so, and hence, will be likely omitted by developers.

Reply:
The policy both answers the questions "where should this #import statement go" and "where would this #import statement be" and it doesn't really impose on developers. Especially for files with a lot of dependencies mandating a deterministic location for the statements can help manage the dependencies quite a bit.

Reply:
> and it doesn't really impose on developers. 

What's the purpose of the policy if nobody follows it?

Reply:
I believe, the alphabetical `#import` policy may only be followed if we would have a linter in place.

Reply:
I am suggesting to skip alphabetical order part at this time and back to this later. Also we need to setup linter on projects.

Reply:
There's plenty of code that doesn't fulfill the existing style guide already, this is a recommendation for both new logic and for whenever existing logic gets altered enough to allow for going an extra step.

Happy to get linter support, but that's a different task and a different debate.